src/DependencyInjection/Configuration.php

```diff
@@ -9,14 +9,14 @@ class Configuration implements ConfigurationInterface
 {
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('hey_update_emoji');
+        $treeBuilder = new TreeBuilder('hey_update_emoji');
+        $rootNode = $treeBuilder->getRootNode();
 
         $rootNode
             ->children()
-                ->scalarNode('image_html_template')
-                    ->defaultValue('<img alt=":{{name}}:" class="emoji" src="https://twemoji.maxcdn.com/36x36/{{unicode}}.png">')
-                ->end()
+            ->scalarNode('image_html_template')
```

Comment:
Can we keep the indentation here, although it's not normal practise, in the case of defining symfony config it helps represent the structure of the config.

Reply:
Done